docs/guide/workspace.md

```diff
@@ -231,6 +231,21 @@ bun test --project e2e --project unit
 ```
 :::
 
+Since Vitest 3.1, you can define a project that will always run even if you filter it out with a `--project` flag. Specify `force: true` flag when defining the project to always include it in your test run (note that this is only available in inline configs):
+
+```ts
+export default defineWorkspace([
+  {
+    // this project will always run even if it was filtered out
+    force: true,
```

Comment:
Same confusion as @hi-ogawa in https://github.com/vitest-dev/vitest/pull/7349#pullrequestreview-2692775570.

Couldn't this same be achieved via plugin that always pushes the project into project filters?

```ts
{
  name: "vitest:always-include-unit-tests",
  configureVitest(context: VitestPluginContext) {
    if(!context.project.vitest.config.project.includes("unit")) {
      context.project.vitest.config.project.push("unit")
    }
  },
},
```

Reply:
> Couldn't this same be achieved via plugin that always pushes the project into project filters?

No, because the filtering happens when the project is resolved. This flag exists to bypass this. See:

https://github.com/vitest-dev/vitest/blob/8ff519a25714aef2d329e33d6452c6a5e30f062f/packages/vitest/src/node/plugins/workspace.ts#L88

Reply:
Hm, no, actually. The source of truth is `_projectFilters`, not `config.project`